Activities/Scripting/UiPath.Scripting.Activities/PowerShell/InvokePowerShellCore.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Activities;
+using System.Activities.Expressions;
+using System.Activities.Statements;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Management.Automation;
+using UiPath.Scripting.Activities.Properties;
+
+namespace UiPath.Scripting.Activities.PowerShell
+{
+    internal class IPSHelper
```

Comment:
Please move this class in a different file.

Reply:
also, add xml comment